src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/MakeBundleTests.cs

```diff
@@ -17,6 +17,7 @@
 namespace Xamarin.Android.Build.Tests
 {
 	[Category ("Node-2")]
+	[Category ("StaticProject")] // TODO: enable for .NET 5
```

Comment:
I believe this test is only validating `BundleAssemblies` behavior, and could use the `MkBundle` category instead?

Reply:
Yeah, I should probably put both categories and can drop the `//TODO:` comment.

`StaticProject` was a category of tests that have static `.csproj` files in our repo. I haven't figured out how these should run under .NET 5 yet.

Reply:
Sounds good, and hopefully this particular instance won't need to be updated if MkBundle remains a legacy feature.